frontend/src/component/common/Table/PaginatedTable/PaginatedTable.tsx

```diff
@@ -28,8 +28,7 @@ const HeaderCell = <T extends object>(header: Header<T, unknown>) => {
             onClick={() => column.toggleSorting()}
             styles={{
                 borderRadius: '0px',
-                paddingTop: 0,
-                paddingBottom: 0,
+                padding: 0,
```

Comment:
previously we had 16px padding from left and right that other components had to compensate for